packages/react-error-overlay/src/index.js

```diff
@@ -47,6 +50,14 @@ export function reportBuildError(error: string) {
   update();
 }
 
+export function reportRuntimeError(
+  error: Error,
+  options?: RuntimeReportingOption = {}
+) {
+  currentRuntimeErrorOptions = options;
+  crashWithFrames(handleRuntimeError)(error);
```

Comment:
Why does this exclude the logic present in the other call?
```js
      if (typeof options.onError === 'function') {
        options.onError.call(null);
      }
```

Reply:
I could move this code 

```js
	try {
      if (typeof options.onError === 'function') {
        options.onError.call(null);
      }
    } finally {
      handleRuntimeError(errorRecord);
    }
```

into `handleRuntimeError`?